client/components/plans/plan/index.jsx

```diff
@@ -51,13 +52,26 @@ module.exports = React.createClass( {
 		siteSuffix = this.props.site ? this.props.site.slug : '';
```

Comment:
we can change this to access `site` directly as it is now extracted from `this.props`